.vsts-ci/templates/releaseBuildAndPushStage.yml

```diff
@@ -9,9 +9,7 @@ stages:
   jobs:
   - template: ./releaseBuildPhase.yml
     parameters:
-      matrix:
```

Comment:
why do we remove the `matrix`? Is it because we no longer use a matrix or that channel is a matrix and we don't need it implied?

Reply:
We had a matrix of one, and I couldn't get it to work in a new scenario.  I just gave up and simplified, TBH

Reply:
got it, that's good to know and I agree I think a matrix of 1 can just be reduced to the channel then.